snow-day/test_banjo.py

```diff
@@ -0,0 +1,13 @@
+"""Testing banjo.py from Codewars."""
+from banjo import are_you_playing_banjo
+
+
```

Comment:
In general, it is bad practice to have a bunch of asserts in one test function.  When something breaks, it's harder to see if other assertions will break after the one that failed.